cs/testing-with-travis.texy

```diff
@@ -0,0 +1,231 @@
+Průběžné testování s Nette Tester
+********************************
+
+/--div .[perex]
+Znáte to. Napsali jste testy, které teď procházejí, poplácali se po zádech a váš spánek začal být mnohem klidnější. Máte jistotu, že váš kód spolehlivě funguje. A pak se to stane. Hodiny ukazují 3:15 a do nočního ticha se zakousne nepříjemný zvuk drnčení mobilu na nočním stolku. Volá šéf. Zase to nefunguje. Zalije vás studený pot. Vždyť na to máme testy! V tu chvíli se vynoří vzpomínka na páteční odpoledne - spěcháte domů a zapomínáte spustit testy po poslední úpravě.
+
+Abyste předešli podobným zážitkům, ukážeme si, jak průběžně testovat pomocí populárního nástroje [Travis CI](https://travis-ci.org/):
+
+- nastavení testovacího prostředí
+- instalace závislostí pomocí Composeru
+- spouštění testů pomocí Nette/Tester
+- nastavení dalších služeb
+- [TL;DR](#Výsledný soubor)
+- integrace s GitHubem
+\--
+
+Jak funguje Travis CI
+=====================
+
+Travis CI (dále jen Travis) je velmi populární nástroj pro automatizaci [průběžné integrace](http://cs.wikipedia.org/wiki/Průběžná_integrace). Je plně integrován do prostředí GitHubu.
+
+Travis se nastavuje pomocí konfiguračního souboru `.travis.yml` umístěného v kořenovém adresáři projektu. Obsahuje několik bloků, které si podrobněji popíšeme.
+
+Bloky jsou spouštěny v pořadí `before_install`, `install`, `before_script`, `script`, `after_success` nebo `after_failure` a nakonec `after_script`. Povinný je pouze blok `script`.
+
```

Comment:
Jen bych to hodil pod sebe na odrážky:
- `before_install`
- ...
- `script` (jediný povinný blok)
- ...
- `after_success` a `after_failure`
- ...